Jenkinsfile

```diff
@@ -121,7 +121,12 @@ Note this is another run, will double the time and no guaranty to have same erro
             archiveArtifacts(artifacts: 'esgf-compute-api-*/examples/*.ipynb', fingerprint: true)
             archiveArtifacts(artifacts: 'PAVICS-landing-*/content/notebooks/climate_indicators/*.ipynb', fingerprint: true)
             archiveArtifacts(artifacts: 'buildout/*.output.ipynb', fingerprint: true)
-            archiveArtifacts(artifacts: 'buildout/env-dump/', fingerprint: true)
+            if (fileExists('buildout/env-dump/')) {
```

Comment:
why check the `env-dump` dir when the error is `No artifacts found that match the file pattern "buildout/*.output.ipynb"` !  It's the line right above that would needed it.

However I propose adding `allowEmptyArchive = true` (https://github.com/jenkinsci/jenkins/blob/1d0e81acc820ea6531b33eac3a630dd10700e16a/core/src/main/java/hudson/tasks/ArtifactArchiver.java#L88-L92) for all of the `archiveArtifacts` instead because we could disable any repos and the other lines could potentially error out as well.

The reason for `No artifacts found that match the file pattern "buildout/*.output.ipynb"` is because the checkbox `SAVE_RESULTING_NOTEBOOK` is unchecked.

Reply:
Oh right! It shows I'm not awake yet. Need more coffee. 
Good idea about `allowEmptyArchive`, I'll adjust with this approach.

Reply:
Not adding to add `archiveArtifacts`?

Reply:
I think it is fair to add it only to optional one. 
Others are expected. Better to flag the workflow failing if they are missing.

Reply:
I was planning to not download the repos that  are not activated in https://github.com/Ouranosinc/PAVICS-e2e-workflow-tests/issues/30 so the other `archiveArtifacts` can potentially raise false error as well.

But sure, we can cross that bridge when we get there.